python_ta/reporters/plain_reporter.py

```diff
@@ -106,6 +107,10 @@ def print_messages(self, level='all'):
         print('\n')
 
     def sort_messages(self):
+        max_messages = float('inf')
```

Comment:
This should be 5, which is the reported default. On the other hand, It should be documented in the sample `.pylintrc` file how to configure pyta to display an unlimited number of messages (by setting the option's value to `float('inf')`?)